### DISCUSSION TITLE
Next.js Website the background svg and the hover effect of that animated grid, how to do that?

### DISCUSSION THREAD

**Discussion by zied-23** [vercel/next.js#74166] [Ideas]
### Impressed By nextjs.org smooth animations

1.animated grid
2.animated svg background
3.animated colorful gradient paths
![bm6DlkCU](https://github.com/user-attachments/assets/c16a3973-3da6-4193-b88d-48424c82ecc5)
![k2Tze7b8](https://github.com/user-attachments/assets/1ae58b91-f909-4cb5-b1cc-7366790cd192)
![jy71CGzF](https://github.com/user-attachments/assets/30f48b5a-4a39-4049-9b68-f6b3ecb5cc01)

### Background

i really got impressed, what package or type styling ad animated should i learn for those animations using Nextjs 15 

### Proposal

is there any example like the one in [Nextjs.org](https://nextjs.org/) i already visited   (https://ui.aceternity.com/) but thats not smooth as the one of Nextjs Website noticing landing page performance and not charged animations, help me get to the right space

**Comment by icyJoseph** [comment]
Those use framer-motion IIRC, https://motion.dev/docs/react-quick-start
